FHIR-us-davinci-deqm.xml

```diff
@@ -1,11 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<specification xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" ballotUrl="http://hl7.org/fhir/us/davinci-deqm/2020Feb" ciUrl="http://build.fhir.org/ig/HL7/davinci-deqm" defaultVersion="3.0.0" defaultWorkgroup="" gitUrl="https://github.com/HL7/davinci-deqm" url="http://hl7.org/fhir/us/davinci-deqm">
+<specification xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" ballotUrl="http://hl7.org/fhir/us/davinci-deqm/2020Feb" ciUrl="http://build.fhir.org/ig/HL7/davinci-deqm" defaultVersion="3.1.0" defaultWorkgroup="cqi" gitUrl="https://github.com/HL7/davinci-deqm" url="http://hl7.org/fhir/us/davinci-deqm">
 <version code="current" url="http://build.fhir.org/ig/HL7/davinci-deqm"/>
+<version code="3.1.0" url="http://hl7.org/fhir/us/davinci-deqm/STU3.1"/>
```

Comment:
We need a line for 4.0.0-ballot (not necessarily in this PR)? Like QI-Core https://github.com/HL7/fhir-qi-core/blob/master/FHIR-us-qicore.xml#L4 
`<version code="4.0.0-ballot" url="http://hl7.org/fhir/us/davinci-deqm/2023Jan"/>`

Reply:
This file is auto-generated. It would be a separate PR to update the ig.json to update the versions.